src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -241,9 +241,14 @@ class editIndex extends viewModelBase {
 
     private updateIndexFields() {
         const map = this.editedIndex().maps()[0].map();
+
         const additionalSourcesDto = {} as dictionary<string>;
         this.editedIndex().additionalSources().forEach(x => additionalSourcesDto[x.name()] = x.code());
-        new getIndexFieldsFromMapCommand(this.activeDatabase(), map, additionalSourcesDto)
+
+        const additionalAssembliesDto: Array<Raven.Client.Documents.Indexes.AdditionalAssembly> = [];
```

Comment:
please use map instead of foreach.

const additionalAssembliesDto = this.editedIndex().additionalAssemblies().map(x => x.toDto());

Reply:
done